lib/ClangImporter/ImportType.cpp

```diff
@@ -2616,12 +2616,10 @@ bool ClangImporter::Implementation::isDefaultArgSafeToImport(
   return true;
 }
 
-static ParamDecl *getParameterInfo(ClangImporter::Implementation *impl,
-                                   const clang::ParmVarDecl *param,
-                                   const Identifier &name,
-                                   const swift::Type &swiftParamTy,
-                                   const bool isInOut,
-                                   const bool isParamTypeImplicitlyUnwrapped) {
+static std::optional<ParamDecl *> getParameterInfo(
+    ClangImporter::Implementation *impl, const clang::ParmVarDecl *param,
+    const Identifier &name, const swift::Type &swiftParamTy, const bool isInOut,
+    const bool isParamTypeImplicitlyUnwrapped, const bool isTemplated) {
```

Comment:
Could we check for `isa<clang::FunctionTemplateDecl>(param->getDeclContext())` instead of having an extra `isTemplated` parameter?

Reply:
Done.